circuits/package.json

```diff
@@ -22,13 +23,11 @@
     "test-tallyVotes": "NODE_OPTIONS=--max-old-space-size=4096 ts-mocha --exit ts/__tests__/TallyVotes.test.ts"
   },
   "dependencies": {
-    "circomlib": "weijiekoh/circomlib#ac85e82c1914d47789e2032fb11ceb2cfdd38a2b",
-    "shelljs": "^0.8.3",
-    "snarkjs": "^0.5.0",
-    "tmp": "^0.2.1"
+    "circomlib": "https://github.com/weijiekoh/circomlib#ac85e82c1914d47789e2032fb11ceb2cfdd38a2b",
+    "snarkjs": "^0.5.0"
   },
   "devDependencies": {
-    "@types/chai": "^4.3.11",
+    "@types/chai": "^4.3.9",
```

Comment:
Why did you downgrade this dep?

Reply:
My bad, messed up while rebasing for sure